src/endpoint/s3/s3_bucket_policy_utils.js

```diff
@@ -205,12 +217,14 @@ async function is_statement_fit_of_method_array(statements, account, method_arr,
 
 async function _is_statements_fit(statements, account, method, arn_path, req) {
     for (const statement of statements) {
+
+        dbg.log1(" statement, account, method, arn_path ", statement, account, method, arn_path);
```

Comment:
I don't think this log is needed, all this information is already logged both by each verifying function wit, with what it compared it to.
If you still think it helps, please change it to something like:
`bucket_policy:is_statements_fit begin validation: statement, account, method, arn_path`

Reply:
Removed.